### ISSUE TITLE
makechr v1.5 does not show tiles on mouseover

### ISSUE THREAD

**Issue by urbster1** [dustmop/makechr#17]
I don't seem to see the blinking box when mousing over tiles in makechr v1.5, but I do see it in makechr v1.4

**Comment by dustmop** [comment]
Hello @urbster1! Apologies for not replying sooner. I'll admit that I didn't do enough work testing the gui app for the 1.5 release, this is definitely a bug, and I believe there are a few others that slipped through as well.

I'm going to try working on makechr this week and next, to release version 1.6 at the end of next week. Could you just let me know, what operating system are you using?

**Comment by urbster1** [comment]
Yep! Using Windows 10 x64 at work, and Windows 7 x64 at home. Let me know
if you need anything else :)

**Comment by dustmop** [comment]
Ok good to know! I have a Windows 10 machine available, I'll make sure it works properly there. Thanks for the feedback.

**Comment by dustmop** [comment]
So, sadly, this didn't happen over the weekend. It's still high on my todo list, and hopefully I'll have time soon to take care of it.
